{"name": "操作日志列表-失败", "status": "passed", "attachments": [{"name": "log", "source": "eb62fb8f-495b-4f06-b9e1-2dcbe416bd50-attachment.txt", "type": "text/plain"}], "start": 1692178396545, "stop": 1692178396597, "uuid": "8de51ab8-d9d4-4927-8d9d-c2fb95da2c51", "historyId": "ad1e9b3def66782ad162eeb06d1cf45f", "testCaseId": "6185372399dbf1b63eb947da37d9a5ce", "fullName": "opsPC.test_logManage#test_pc_log_list_002", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "日志管理"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_logManage"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "13268-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_logManage"}]}